apps/challenges/serializers.py

```diff
@@ -23,7 +23,7 @@ class ChallengeSerializer(serializers.ModelSerializer):
     def __init__(self, *args, **kwargs):
         super(ChallengeSerializer, self).__init__(*args, **kwargs)
         context = kwargs.get("context")
-        if context and context.get("request").method != "GET":
+        if context and context.get("request").method == "POST":
```

Comment:
Let's not do these changes in this PR. Can you please open up a new PR with backend changes only.